community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -375,47 +374,7 @@
                   "the page cache. The default page cache memory assumes the machine is dedicated to running " +
                   "Neo4j, and is heuristically set to 50% of RAM minus the max Java heap size." )
```

Comment:
I'll update the text and make the default case more vague, so as to distance the setting from how the computation of the default page cache memory is implemented. This makes the documentation of the setting, only be about the setting.